components/ChallengeMaterial.tsx

```diff
@@ -19,17 +23,21 @@ type ChallengeMaterialProps = {
 
 export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
   const { currentState: state, active } = props
-  const cardStyles = []
+  const cardStyles = ['challenge-title-card']
   if (state === 'complete') {
-    cardStyles.push('challenge-title-card--done')
+    cardStyles[0] += '--done'
   } else {
     cardStyles.push('shadow-sm', 'border-0')
   }
   if (active) {
-    cardStyles.push('challenge-title-card--active')
+    cardStyles[0] += '--active'
   }
   return (
-    <div className={`card mb-2 ${cardStyles.join(' ')}`}>
+    <div
+      data-testid="challenge-title"
```

Comment:
What is data-testid?

Reply:
It's for tests --> [React Testing Library](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid)